src/09-coveio_abi.adoc

```diff
@@ -369,38 +417,73 @@ The possible error codes returned in `sbiret.error` are shown below.
 | SBI_ERR_FAILED          | The operation failed for unknown reasons.
 |===
 
-[#sbi_covg_get_device_certificate]
-===== Function: CoVE Guest Get Device Certificate (FID #1025)
+[#sbi_covg_get_device_connection_transcript]
+===== Function: CoVE Guest Get Device Connection Transcript (FID #1025)
 [source, C]
 -----
-struct sbiret sbi_covg_get_device_certificate(unsigned long device_if_id,
-                                                       unsigned long slot_id,
-                                                       unsigned long cert_addr_out,
-                                                       unsigned long cert_size);
+struct sbiret sbi_covg_get_device_connection_transcript(unsigned long device_if_id,
+                                                        unsigned long transcript_element_id,
+                                                        unsigned long transcript_element_parameter,
+                                                        unsigned long data_addr_out,
+                                                        unsigned long data_size);
 -----
 
-Get the certificate chain for the physical device hosting `device_if_id`
-based on `slot_id`.
-
-The TSM returns the certificate chain in the form of the <<SPDM>> defined
-`certificate chain format`.
+Get a specific element from the device connection transcript, or the whole
+transcript.
 
-The TVM calls this function in order to verify the authenticity of the physical
-device. A TVM must not accept a device interface without doing that verification
-first.
+The `transcript_element_id` specifies which part of the device connection
+transcript the TVM is requesting. It must be one of the following values:
 
-`slot_id` must be between 0 and 7 inclusive.
+[source, C]
+-----
+enum sbi_covg_transcript_element {
+     /* Full device connection transcript */
+     FULL_TRANSCRIPT = 0,
+
+     /*
+      * Reconstructed certificate chain for the physical device hosting
+      * `device_if_id`, from the transcript `CERTIFICATE` SPDM responses.
+      * The TSM returns the certificate chain in the form of the <<SPDM>>
+      * defined `certificate chain format`.
+      * The `transcript_element_parameter` is set to a certificate chain slot
+      * ID and thus must be between 0 and 7 inclusive.
+      */
+     DEVICE_CERTIFICATE = 1,
+
+     /*
+      * Reconstructed device measurement for the physical device hosting
+      * `device_if_id`, from the transcript `MEASUREMENTS` SPDM responses.
+      */
+     DEVICE_MEASUREMENT = 2,
+
+     /*
+      * The `CAPABILITIES` SPDM response for the physical device hosting
+      * `device_if_id`, from the device connection transcript.
+      */
+     SPDM_CAPABILITIES = 3,
```

Comment:
why we need `SPDM_CAPABILITIES`, if we already have VCA?

Or if we do want to have `SPDM_CAPABILITIES`, then I recommend we return `SPDM_VERSION` and `SPDM_ALGORITHMS` as well.

Reply:
I removed it and only kept `VCA`.